lib/cisco_node_utils/cmd_ref/pim.yaml

```diff
@@ -21,6 +21,12 @@ all_ssm_ranges:
   multiple:
   get_value : '/^<afi> pim ssm range (.*)$/'
 
+bfd:
+  kind: boolean
+  get_value: '/^ip pim bfd$/'
```

Comment:
To @chrisvanheuveln point about ipv6 support, this should be.

`get_value: '/^<afi> pim bfd$/'` and
`set_value: "<state> <afi> pim bfd"`